data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,394 @@ mission "Wanderers: Pre-Colony"
+			`	Something of an argument is taking place in front of a tent at the edge of the landing area. There are a lot of wings flapping and raised voices, but they aren't speaking Hai, so you have no idea what it's about.`
+			choice
+				`	(Go over and investigate.)`
+					goto investigate
+				`	(Leave and explore this part of space.)`
+					decline
+					
+			label investigate
+			`	The argument seems to be breaking up, with unhappy captains heading back to their ships. You see a few Tempests powering up for departure.`
+			`	You walk up to the Wanderer they were all shouting at. He looks at you warily as you greet him in Hai. He is briefly startled and cocks his head and regards you for a moment before saying, "Ah. You are the [lost, lonely] one from from the worlds beyond the Hai." He holds up a taloned hand and continues before you can say anything. "Spaceport construction hasn't even started. I don't have any Thunderheads. I don't have any parts. I don't have any fuel. If you need something, you have to go back to <destination>. I don't know when it will get better because everyone is [dithering, flying around] and no one is [available, willing] to do transport work."`
+			choice
+				`	"Tell me what you need and I'll handle it."`
+					goto handle
+				`	"I guess I'll just leave, then."`
+					decline
+
+			label handle
+			`	"[Truly? Honestly?]" he asks. When you nod your head he hands you a data chip and says, "Take this to Vara K'chrai and they will make the arrangements. We need construction equipment, materials, and workers. I can't request them directly since communications don't work through the wormhole.`
+			`	You turn to go as a group of Wanderers with surveying equipment walks up. They are all dressed in hard hats and safety gear. The construction supervisor you've been speaking to pulls up a hologram of the site to discuss with the surveyors. As you walk off, the translator just picks up, and you hear him muttering to himself. "The Eye opens and we are getting help from the [alien, stranger] and not our own flock."`
+				accept
+
+	on decline
+		event "wanderers: spera anatrusk colony" 40
+
+	on abort
+		event "wanderers: spera anatrusk colony" 40
+
+	on fail
+		event "wanderers: spera anatrusk colony" 40
+
+	on complete
+		conversation
+			`As you land at the spaceport, you're sure you see a few of the ships you saw on <planet> with various outfits stacked for installation. It's hard to find anyone who speaks Hai, but you find your way to the main spaceport office and hand over the chip. The person at the desk chitters excitedly and calls someone over. They quickly enter information into their systems and then hand the chip back to you. After a few moments of confusion where you end up making your Hai ear gesture, they access a device and speak into it. Your translator picks up "Job board. Job board." Hopefully everything has been booked for you.`
```

Comment:
> "Job board. Job board."

This sounds like we're being directed to the job board to pick up jobs, but the followup missions are already accepted on landing. Maybe something more along the lines of "Mission list"?